staging/src/k8s.io/kubectl/pkg/cmd/wait/wait_test.go

```diff
@@ -574,26 +574,46 @@ func TestWaitForDeletion(t *testing.T) {
 			timeout: 10 * time.Second,
 
 			validateActions: func(t *testing.T, actions []clienttesting.Action) {
-				if len(actions) != 6 {
+				// Due to test flakiness this needs to allow for 5 to 6 actions
+				if len(actions) < 5 && len(actions) > 6 {
```

Comment:
how can len(actions) be less than 5 and greater than 6 at the same time :smile: ?

Reply:
Whoops, thanks for catching that. I think I'm just gonna strip out the validate action testing honestly. With the move to informers the number of actions taken becomes variable, and the number of potential valid cases to test for spirals out of control very fast.